### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Deploy Web App_Keyboard: Focus is lost after tabbing from ‘Deploy to Root’ checkbox

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3336]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.4.3

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till deploy to Root checkbox and press tab
11) Verify is focus is lost after tabbing from ‘Deploy to Root’ checkbox

**Actual Result:** 
Focus is lost after tabbing from ‘Deploy to Root’ checkbox

**Expected Result:** 
Keyboard focus should not be lost when tabbing through the popup. From Deploy to root checkbox focus should go to the links present in the app service details pane

**User Impact:** 
Keyboard users will find it confusing about navigating in the popup if there is any focus loss

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}

[Bug_14_Focus is lost after deploy to root checkbox.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451249/Bug_14_Focus.is.lost.after.deploy.to.root.checkbox.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati  this issue fixed it #3507, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3773/Azure/processDownloadRequest/artifacts/WindowsAzurePlugin4EJ.PR.3773.09-09-2019.zip), please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
@Flanker32
Regressed in the above build but the issue is not fixed. Focus is lost after tabbing from ‘Deploy to Root’ checkbox. Attached latest video for reference.
[Issue not fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3594056/Issue.not.fixed.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati Here is the [latest build](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_develop/1250/Azure/) which fix this issue, could you please help us review it again? Thanks a lot!

**Comment by Tanushagottipati** [comment]
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
Issue fixed by downloading above build. hence closing the bug and attached latest video for reference
[Uploading issue fixed.pptx…]()

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749801
